components/doc.tsx

```diff
@@ -308,19 +307,20 @@ function SideBarHeader({ children }: { children: Child<string> }) {
       : undefined;
     let title = module;
```

Comment:
Nitpick: consider extracting this out ot a function (ex. `const { title, subtitle } = getTitles(parsed)`)